viite-UI/src/less/core/forms.less

```diff
@@ -287,10 +287,19 @@ form.roadAddressProject .control-label-small {
   margin-right: 11px;
   margin-top: 30px;
   margin-bottom: -10px;
-  width: 55px !important;
+  width: 45px !important;
```

Comment:
I'm afraid that modifying classes already in use by other forms : just to adapt to a newly created form is not a good idea.
We should have one or two classes : junction / junction point and there we define all the rules we need.

Even if the rules are the same, it will be alot easier to modify them only for one specific form in the future.

Reply:
Fixed and pushed to Git.